src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const rentalCost = 40;
+  const finalCost = rentalCost * days;
+
+  if (days < MEDIUM_TERM) {
```

Comment:
I think it's better to check for a long term, then for medium term and then just return basic price without any discounts